docs/client-specifications/configuration.md

```diff
@@ -0,0 +1,200 @@
+<img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
+
+# Momento Client SDKs Specification: Client Configuration
+
+This document provides information on how Momento client implementations must expose configuration options to users.
+
+All code in this example repo is TypeScript pseudocode.  Implementations should use the appropriate idioms for the
+programming language at hand.
+
+# Tenets
+
+The overarching goals of the specification are:
+
+* Configuration should be as simple as possible for users.  We do testing and tuning work up front to make sure
+  have established high-confidence default settings that cover the 90% use case, so that users don't have to spend
+  time figuring it out themselves.
+* Because there are different requirements for different environments (e.g. dev vs. prod), we provide a small handful of
+  pre-built configurations that users can choose from, out-of-the box.
+* Configurations must be specified as interfaces, so that alternate implementations can be added over time.
+* Users should always have a last-resort option of manually tuning the settings themselves, in case there are special
+  cases that our defaults are not quite right for.
+
+# Choosing a configuration
+
+The `SimpleCacheClient` constructor should accept a required argument whose type is our `Configuration` interface.  e.g.:
+
+```typescript
+import * as config from '@gomomento/sdk/config';
+
+const client = SimpleCacheClient({config: config.Laptop.latest()});
```

Comment:
I like the config.Laptop seeing it used here now. I think is better then `Dev` for sure